### DISCUSSION TITLE
Is it possible to run MD with other Hamiltonians?

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#725] [Q&A] [answered]
Hello,

From the  xTB [page](https://xtb-docs.readthedocs.io/en/latest/md.html#molecular-dyamics-simulations) we read:

`For MD simulations with GFN-FF the time step must be reduced, for more information see section [GFN-Force-Field (GFN-FF)](https://xtb-docs.readthedocs.io/en/latest/gfnff.html#gfnff)`

Does this mean that I can run MD with GFN2-xTB, for example? If yes, how?

**Comment by cplett** [comment, marked as answer]
Hi,
with `xtb` it is possible to run MD and MTD simulations with GFN2-xTB, GFN1-xTB, and GFN-FF. GFN2-xTB is currently the default, but you can always select the method by adding the respective flag in your command line call (`--gfn2`, `--gfn1`, and `--gfnff`).
